impress-org/givewp
# Epic Feature: Donor and Form table styles

## Description

This PR contains updated styles applied to the Interweave rendered html tags for both the Donor and the form tables. 

## Affects
Donations>Donors & Donations>AllForms

## Visuals
## Old Donor Table:
<img width="1725" alt="old donor table" src="https://user-images.githubusercontent.com/75056371/201572939-e942b264-b75e-4f34-9a23-3ca16d868721.png">

## New Donor Table:
<img width="1725" alt="new donor table" src="https://user-images.githubusercontent.com/75056371/201573601-beb3940a-40d6-4ce1-ab08-8e468c04583d.png">

## Old Form Table:
<img width="1725" alt="Old donation form table" src="https://user-images.githubusercontent.com/75056371/201572935-8a581b6d-7b0b-4eb9-830b-3890ea5becf5.png">

## New form table:
<img width="1725" alt="new form table" src="https://user-images.githubusercontent.com/75056371/201573212-2daf74f5-1cd6-4076-8334-ff883f03299a.png">

## Testing Instructions

1. Visit the Donor table to verify.
2. Visit the All Forms table. (Can not be tested until the Forms model has been created.
## Pre-review Checklist
